pty/src/com/jediterm/pty/PtyMain.java

```diff
@@ -86,4 +97,46 @@ public void write(byte[] bytes) throws IOException {
       super.write(bytes);
     }
   }
+
+  public static class PtyTabChangeListener implements TabChangeListener {
```

Comment:
This class should be definitely put out of the PtyMain to be used from other entry points, like JediTermMain or JBTerminalWidget in IntelliJ